src/react/databrowser/buckets/ObjectLockRetentionSettings.tsx

```diff
@@ -140,62 +146,60 @@ export default function ObjectLockRetentionSettings(props: {
             period.
           </SmallerText>
         </F.Fieldset>
-        <F.Fieldset direction="row" alignItems="baseline">
+        <F.Fieldset direction="row" alignItems="center">
           <F.Label>Retention period</F.Label>
 
-          <div>
-            <Controller
-              control={control}
-              id="retentionPeriod"
-              name="retentionPeriod"
-              render={({ field: { onChange, value: retentionPeriod } }) => {
-                return (
-                  <Input
-                    name="retentionPeriod"
-                    value={retentionPeriod}
-                    onChange={(e) => onChange(e.target.value)}
-                    type="number"
-                    style={{
-                      width: spacing.sp40,
-                    }}
-                    min={1}
-                    disabled={!isDefaultRetentionEnabled}
-                  />
-                );
-              }}
-            />
-          </div>
+          <Controller
+            control={control}
+            id="retentionPeriod"
+            name="retentionPeriod"
+            render={({ field: { onChange, value: retentionPeriod } }) => {
+              return (
+                <Input
+                  name="retentionPeriod"
+                  value={retentionPeriod}
+                  onChange={(e) => onChange(e.target.value)}
+                  type="number"
+                  style={{
+                    width: '4.5rem',
+                    boxSizing: 'border-box',
+                    height: '2.286rem',
+                  }}
+                  min={1}
+                  disabled={!isDefaultRetentionEnabled}
+                />
+              );
+            }}
+          />
 
-          <SpacedBox
-            style={{
-              width: '25%',
+          <Controller
+            control={control}
+            id="retentionPeriodFrequencyChoice"
+            name="retentionPeriodFrequencyChoice"
+            defaultValue={'DAYS'}
+            render={({
+              field: { onChange, value: retentionPeriodFrequencyChoice },
+            }) => {
+              return (
+                <SpacedBox
+                  style={{
+                    width: '25%',
+                  }}
+                  ml={8}
+                >
```

Comment:
Please use Box component from core-ui and check the stories of how to use this component. 
e.g: `<Box width = '25%' ml={8} />`